### PR TITLE
Clover cleanup

### PR DESCRIPTION
**Pull request by bustarhymes69**
Updates minor items for 11-leaf clover instead of ten-leaf/disassembled.  Added a check that we can use ring of conflict while I was there.

### CODE REVIEW

**Review by Malibu-Stacey — DISMISSED** [review]
ignore this (or else!)

File: Source/relay/TourGuide/Items of the Month/2019/Rune Spoon.ash [hunk 1]
```diff
@@ -30,7 +30,7 @@ void IOTMRuneSpoonGenerateResource(ChecklistEntry [int] resource_entries)
             
             "Platypus":"+5 familiar weight and canadia access",
             "Opossum":"+5 adventures/day from food and canadia access",
-        	"Marmot":"+1 extra clover/day and canadia access",
+        	"Marmot":"+1 all resistance and canadia access",
```
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2019/Rune Spoon.ash:33]: RIP to marmot, you were a real one, pouring one out 🍾

File: Source/relay/TourGuide/Pulls.ash [hunk 2]
```diff
@@ -489,13 +489,14 @@ void generatePullList(Checklist [int] checklists)
             pullable_item_list.listAppend(GPItemMake($item[xiblaxian stealth vest], "-combat shirt"));
         if ($item[duonoculars].item_is_usable())
 	        pullable_item_list.listAppend(GPItemMake($item[duonoculars], "-combat, +5 ML"));
-        pullable_item_list.listAppend(GPItemMake($item[ring of conflict], "-combat"));
+        if ($item[ring of conflict].item_is_usable())
```
Comment (docrostov) [inline Source/relay/TourGuide/Pulls.ash:492]: this is not really relevant to this specific PR but i wonder if there is some possible use in eventually disambiguating this to add item_is_usable() more broadly to almost all of this stuff. feels like a good possibility that tptb will keep adding paths that randomly can't use certain shit.

File: Source/relay/TourGuide/Pulls.ash [hunk 3]
```diff
@@ -489,13 +489,14 @@ void generatePullList(Checklist [int] checklists)
...
+            pullable_item_list.listAppend(GPItemMake($item[ring of conflict], "-combat"));
```
Comment (docrostov) [inline Source/relay/TourGuide/Pulls.ash:493]: actually, speaking of which, it looks like this code doesn't actually handle TCRS at all; any equipment pull recs for 2CRS that are pull ideas dependent on enchantments are just straight broken. not relevant to this PR but adding this note to issue #40

File: Source/relay/TourGuide/Quests/Level 11 - Desert.ash [hunk 4]
```diff
@@ -79,7 +79,7 @@ void QLevel11DesertGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEnt
         if (__last_adventure_location == $location[the arid, extra-dry desert]) {
             string [int] description;
             description.listAppend("Adventure in the Oasis.");
-            if ($items[ten-leaf clover, disassembled clover].available_amount() > 0)
```
Comment (docrostov) [inline Source/relay/TourGuide/Quests/Level 11 - Desert.ash:82]: RIP to disassembled clovers, you were a real one, pouring one out 🍾

**Review by docrostov — APPROVED** [review]
thanks for posting these all up busta. good changes all around. will merge in tomorrow or the next day -- would love your thoughts re: my comment on pulls.ash line 492 regarding item_is_usable() being used more broadly.

**Pull request merged**